frontend/src/components/NavBar.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Link } from 'react-router-dom';
+import { Alignment, Button, Navbar, NavbarGroup, Drawer, Position } from '@blueprintjs/core';
+import { deviceSize } from '../theme';
+import Logo from '../resources/medshare-logo.png';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5rem;
+`;
+
+const TopDrawer = styled.div`
+  margin: 1rem;
+`;
+
+const TextWrapper1 = styled.div`
+  margin-left: 2rem;
+  display: flex;
+  flex-direction: column;
+  height: 500px;
+`;
+
+const TextWrapper2 = styled.div`
+  display: flex;
+  border-top-style: solid;
+  align-items: center;
+  justify-content: center;
+  height: 108px;
```

Comment:
I don't think you need to modify the height attribute here. Try using padding instead to achieve the same visuals.

Reply:
done